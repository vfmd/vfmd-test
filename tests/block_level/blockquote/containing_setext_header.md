> This is a blockquote
> that contains
>
> setext header
> =============
> setext header
> -------------
>
> This is part of the blockquote too
