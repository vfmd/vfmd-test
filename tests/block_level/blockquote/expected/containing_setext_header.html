<blockquote>
  <p>This is a blockquote
that contains</p>

<h1>setext header</h1>

<h2>setext header</h2>

<p>This is part of the blockquote too</p>
</blockquote>
